### DISCUSSION TITLE
Welcome to mahdshadivakhande Discussions!

### DISCUSSION THREAD

**Discussion by shadivakhande1390** [shadivakhande1390/mahdshadivakhande#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shadivakhande1390** [comment]
⭐⭐⭐⭐⭐
خیلی عالیه

**Reply by shadivakhand** [reply]
> ⭐⭐⭐⭐⭐
> خیلی عالیه

**Comment by shadivakhande90** [comment]
> 
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
>
